specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/examples/hybridRunbookWorkerGroupUpdate.json

```diff
@@ -0,0 +1,33 @@
+{
+    "parameters": {
+        "subscriptionId": "subid",
+        "resourceGroupName": "rg",
+        "automationAccountName": "testaccount",
+        "hybridRunbookWorkerGroupName": "TestHybridGroup",
+        "api-version": "2015-10-31",
+        "HybridRunbookWorkerGroupUpdateParameters":{
```

Comment:
This is defined as ```parameters``` in the schema (line 175) and is the cause of the model validation failure [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/357626837).

Reply:
Fixed this. Thanks.